nipype/interfaces/dtitk/registration.py

```diff
@@ -24,35 +36,34 @@ class RigidInputSpec(CommandLineInputSpec):
                              default_value=4)
     ftol = traits.Float(mandatory=True, position=6, argstr="%s",
                         desc="cost function tolerance", default_value=0.01)
-    useInTrans = traits.Float(mandatory=False, position=7, argstr="%s",
+    useInTrans = traits.Float(position=7, argstr="%s",
                               desc="to initialize with existing xfm set as 1",
                               default_value=1)
 
 
 class RigidOutputSpec(TraitedSpec):
-    out_file = traits.File(exists=True)
-    out_file_xfm = traits.File(exists=True)
+    out_file = File(exists=True)
+    out_file_xfm = File(exists=True)
 
 
 class RigidTask(CommandLineDtitk):
+    """Performs rigid registration between two tensor volumes
+
+    Example
+    -------
+
+    >>> import nipype.interfaces.dtitk as dtitk
+    >>> node = dtitk.RigidTask()
+    >>> node.inputs.fixed_file = 'diffusion.nii.gz'
+    >>> node.inputs.moving_file = 'diffusion2.nii.gz'
+    >>> node.inputs.similarity_metric = 'EDS'
+    >>> node.inputs.samplingX = 4
+    >>> node.inputs.samplingY = 4
+    >>> node.inputs.samplingZ = 4
+    >>> node.inputs.ftol = 0.01
+    >>> node.inputs.useInTrans = 1
+    >>> node.run() # doctest: +SKIP
```

Comment:
Could you add a `node.cmdline` with the expected command line, here?

Similarly, this would be good to add to any of the docstrings that are currently missing it.